src/styles/index.scss

```diff
@@ -1,3 +1,144 @@
-body {
+.page {
+  scroll-behavior: smooth;
+  background: #fff;
+  font-family: Roboto, sans-serif;
+
+  &_body {
+    margin: 0;
+  }
+}
+
+.card {
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  width: 198px;
+  padding-left: 16px;
+  padding-right: 16px;
+  box-sizing: border-box;
+}
+
+img {
```

Comment:
Consider using a class selector for styling

Reply:
This comment I didn't understand too